src/components/ManageAccountsDrawer/components/LedgerOpenApp.tsx

```diff
@@ -0,0 +1,134 @@
+import { Button } from '@chakra-ui/react'
+import {
+  arbitrumChainId,
+  arbitrumNovaChainId,
+  avalancheChainId,
+  bchChainId,
+  binanceChainId,
+  bscChainId,
+  btcChainId,
+  type ChainId,
+  cosmosChainId,
+  dogeChainId,
+  ethChainId,
+  gnosisChainId,
+  ltcChainId,
+  optimismChainId,
+  polygonChainId,
+  thorchainChainId,
+} from '@shapeshiftoss/caip'
+import type { slip44Table } from '@shapeshiftoss/hdwallet-core'
+import { isLedger } from '@shapeshiftoss/hdwallet-ledger'
+import { useCallback, useEffect } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { selectFeeAssetByChainId } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { DrawerContentWrapper } from './DrawerContent'
+
+export type LedgerOpenAppProps = {
+  chainId: ChainId
+  onClose: () => void
+  onNext: () => void
+}
+
+type Slip44Key = keyof typeof slip44Table
+const getSlip44KeyFromChainId = (chainId: ChainId): Slip44Key | undefined => {
+  switch (chainId) {
+    // UTXO chains
+    case btcChainId:
+      return 'Bitcoin'
+    case dogeChainId:
+      return 'Dogecoin'
+    case bchChainId:
+      return 'BitcoinCash'
+    case ltcChainId:
+      return 'Litecoin'
+    // EVM chains
+    case ethChainId:
+      return 'Ethereum'
+    case avalancheChainId:
+      return 'Avalanche'
+    case optimismChainId:
+      return 'Optimism'
+    case bscChainId:
+      return 'BnbSmartChain'
+    case polygonChainId:
+      return 'Polygon'
+    case gnosisChainId:
+      return 'Gnosis'
+    case arbitrumChainId:
+      return 'Arbitrum'
+    case arbitrumNovaChainId:
+      return 'ArbitrumNova'
+    // Cosmos chains
+    case thorchainChainId:
+      return 'Rune'
+    case cosmosChainId:
+      return 'Atom'
+    case binanceChainId:
+      return 'Binance'
+    default:
+      return undefined
+  }
+}
+
+export const LedgerOpenApp = ({ chainId, onClose, onNext }: LedgerOpenAppProps) => {
+  const translate = useTranslate()
+  const asset = useAppSelector(state => selectFeeAssetByChainId(state, chainId))
+  const maybeLedgerWallet = useWallet().state.wallet
+  const wallet = maybeLedgerWallet && isLedger(maybeLedgerWallet) ? maybeLedgerWallet : undefined
```

Comment:
Same thing here - could check for `if (!(wallet && isLedger(wallet))` dance in `isCorrectAppOpen` which is the only consumer

Reply:
https://github.com/shapeshift/web/pull/6779/commits/fa0ec15b703f1d063bdc1e91a7689a4d7ca79295